Tests/interop/net/method/test_arguments.py

```diff
@@ -268,7 +269,7 @@ def test_default_values_2(self):
 
         f(1, **{'y':2}); Flag.Check(3)
 
-        # public void M320([DefaultParameterValue(40)] int y, int x) { Flag.Reset(); Flag.Set(x + y); }
+        # public void M320([DefaultParameterValue(40)] int y, int x) { Flag.Set(x + y); }
```

Comment:
Might be nice to change the definition in the DLR to:
```
public void M320([DefaultParameterValue(40)] int x, int y) { Flag.Set(x * 10 + y); }
```

Reply:
Totally agree. Will do it in one of the upcoming PRs for the DLR.